privacyidea/lib/resolver.py

```diff
@@ -364,6 +378,16 @@ def pretestresolver(resolvertype, params):
     :type params: dict
     :return:
     """
+    # If an already saved resolver is tested again, the password
+    # could be "__CENSORED__". In this case we use the old, saved password.
+    if params.get("resolver"):
+        old_config_list = get_resolver_list(filter_resolver_name=params.get("resolver")) or {}
+        old_config = old_config_list.get(params.get("resolver")) or {}
+        for key in old_config.get("__CENSOR_KEYS__"):
```

Comment:
The config should always contain the CENSOR_KEYS. These are loaded right at the beginning in the config object here:
https://github.com/privacyidea/privacyidea/blob/1271/hide-ldap-password/privacyidea/lib/config.py#L128

Reply:
Ah okay, but I just realized we should still have ``for key in old_config.get("__CENSOR_KEYS__", [])`` here in case the resolver given in the ``resolver`` parameter does not exist.

Reply:
What is the corner case you are thinking of here?

Reply:
done in f853a40d

Reply:
:+1: